federation/pkg/kubefed/init/init.go

```diff
@@ -159,6 +160,7 @@ func (o *initFederationOptions) Bind(flags *pflag.FlagSet, defaultServerImage, d
 	flags.StringVar(&o.dnsProviderConfig, "dns-provider-config", "", "Config file path on local file system for configuring DNS provider.")
 	flags.StringVar(&o.etcdImage, "etcd-image", defaultEtcdImage, "Image to use for etcd server.")
 	flags.StringVar(&o.etcdPVCapacity, "etcd-pv-capacity", "10Gi", "Size of persistent volume claim to be used for etcd.")
+	flags.StringVar(&o.etcdPVStorageClass, "etcd-pv-storage-class", "", "The storage class of the persistent volume claim used for etcd.   Must be provided if a default storage class is not enabled for the host cluster.")
```

Comment:
This is a change in behavior from the alpha annotation, and I want to make sure you're aware and making the decision consciously: If a user runs without specifying `etcd-pv-storage-class` and their cluster doesn't have a default storage class installed this will result in the PVC not provisioning a new volume and instead waiting to bind to an existing PV, whereas the alpha annotation always provisioned a new volume.